portality/bll/services/journal.py

```diff
@@ -202,7 +206,8 @@ def _get_doaj_meta_kvs(journal):
                 ("DOAJ Seal", YES_NO.get(journal.has_seal(), "")),
                 # ("Tick: Accepted after March 2014", YES_NO.get(journal.is_ticked(), "")),
                 ("Added on Date", journal.created_date),
-                ("Last updated Date", journal.last_manual_update)
+                ("Last updated Date", journal.last_manual_update),
+                ("OA Start Date", journal.bibjson().oa_start)
```

Comment:
This is going to make the public journal CSV expose the OA Start Date, which we don't want to do yet.  We only want that available in the admin csv for the moment, so we need to move this to the other method.

Reply:
I think I figured that out now!